packages/manager/modules/netapp/src/dashboard/network-configuration/controller.js

```diff
@@ -85,6 +85,13 @@ export default class OvhManagerNetAppNetworkConfigurationCtrl {
       this.selectedSubnet = noSubnet;
       this.disableSubnetField = true;
     } else {
+      this.subnets.map((subnet) => {
+        const newSubnet = subnet;
+        if (!newSubnet.displayName) {
+          newSubnet.displayName = subnet.cidr;
+        }
+        return newSubnet;
+      });
```

Comment:
Since `.map` does not modify the array but return a new array, I don't think this does anything